src/coreclr/minipal/Unix/symbols.cpp

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
```

Comment:
Banner

Reply:
Thank you for spotting the missing headers